### PR TITLE
Deployed tag

### PR DESCRIPTION
**Pull request by JulianWielga**
Zrzut ekranu 2020-09-24 o 19 06 32

### CODE REVIEW

File: ui/client/components/HistoryItem.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import {css, cx} from "emotion"
+import React, {useMemo} from "react"
+import {useTranslation} from "react-i18next"
+import styles from "../stylesheets/processHistory.styl"
+import Date from "./common/Date"
+import {ReactComponent as Badge} from "./deployed.svg"
+import {ActionType, ProcessVersionType} from "./Process/types"
+
+type HistoryItemProps = {
+  isLatest?: boolean,
```
Comment (lciolecki) [inline ui/client/components/HistoryItem.tsx:10]: Hmm, as I remeber each object has isLatest and isDeploted props so you can remove option `?`

File: ui/client/test/ProcessHistory-test.js [hunk 2]
```diff
@@ -1,19 +1,32 @@
-import React from 'react'
+import Enzyme, {mount} from "enzyme"
```
Comment (lciolecki) [inline ui/client/test/ProcessHistory-test.js:1]: Nice ;)

**Review by lciolecki — APPROVED** [review]
LGTM, but please look at comments.

File: ui/client/components/HistoryItem.tsx (hunk 1 above)
Comment (JulianWielga) [reply ui/client/components/HistoryItem.tsx:10]: what you are talking about?

**Pull request merged**